go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/TecharoHQ/anubis
 
-go 1.24.2
+go 1.25.5
```

Comment:
It would make it easier to package the software for distros, if not a patch level would be tracked here. Bumping Go usually requires many rebuilds and running the software with 1.25.4 for a week works.